Sources/MediaViewer/UI/PreviewController.swift

```diff
@@ -208,6 +230,11 @@ extension PreviewController {
             .topViewController?
             .view
     }
+    
+    // returns the current view controller viewed by the page vc
```

Comment:
But why? I can see what the function is doing, but why?

Reply:
this is used during the dismiss transition, to get the view of the currently being dismissed controller.